### PR TITLE
Removed org.freedesktop.secrets dependency and removed GNOME category.

### PR DESCRIPTION
**Pull request by sarvasana**
People use this on KDE as well.

### CODE REVIEW

**Comment by Azd325** [comment]
What is your reason for removing `org.freedesktop.secrets`?

**Comment by sarvasana** [comment]
Because it is not a dependency of GitKraken and forces KDE users to install GNOME Keyring or KeepassXC.

File: gitkraken/GitKraken.desktop [hunk 1]
```diff
@@ -6,5 +6,5 @@ Icon=gitkraken
 Terminal=false
 Type=Application
 StartupNotify=true
-Categories=GNOME;GTK;Development;Documentation;
```
Comment (Azd325) [inline gitkraken/GitKraken.desktop:9]: This category is from the upstream `.desktop` file, so I want to keep this.
Also this a related category to GTK. https://specifications.freedesktop.org/menu-spec/latest/apas02.html

File: gitkraken/PKGBUILD [hunk 2]
```diff
@@ -16,7 +17,7 @@ url="https://www.gitkraken.com/"
 provides=('gitkraken')
 arch=('x86_64')
 license=('custom')
-depends=('nss' 'libxtst' 'org.freedesktop.secrets' 'alsa-lib' 'libxss')
```
Comment (Azd325) [inline gitkraken/PKGBUILD:19]: Can you add this as `optdepends` because the Debian Package offers this as a suggestion package?

**Review by Azd325 — CHANGES_REQUESTED** [review]
Thanks for PR
I rechecked upstream and if `org.freedesktop.secrets` is defined as optional dependency I am happy to merge it.

**Pull request closed without merging**